Copy edit and suggestions for the writing handbook (#1082)
GoogleChrome/web.dev: src/site/content/en/handbook/audience/index.md

description: |

>   Definitions of the the key personas for web.dev and advice about how to write for them.

Author: I noticed that the descriptions are inconsistently written. Should it be a complete sentence, description (like alt text), or actionable (Learn how to write effective docs!). Is there guidance for this? There probably should be, since this is in every doc, right?

Reviewer: The description field is going into the <meta> description field and will show up in search results. My hunch is we want it to be complete sentences.

Author: "A meta description tag should generally inform and interest users with a short, relevant summary of what a particular page is about. They are like a pitch that convince the user that the page is exactly what they're looking for. There's no limit on how long a meta description can be, but the search result snippets are truncated as needed, typically to fit the device width."

More info here: https://support.google.com/webmasters/answer/35624

Reviewer 2: My impulse here is to provide guidelines (probably more or less the bit Lizzi quoted) plus a few templates people can choose from (e.g., "Learn to do x with y"). Also, seems to me this could be either a sentence or a noun phrase? But I would happily defer to others' expertise :)

Reviewer 3: I think brevity (read 'pithy') is best here. Noun phrases should at least be allowed and that they are allowed should be communicated.

Reviewer 2: Added note in Google Doc to incorporate this guidance.

---

> When you're writing for web.dev, it's important to consider the key personas so that each piece you write helps readers learn or accomplish something.

Author: Need a value statement for this doc

Reviewer 2: ```suggestion
When you're writing for web.dev, it's important to consider the persona or personas you're writing for. Keeping your audience in mind helps you make good writing decisions, like how much background information to include or whether you need to define a key term. Those considerations help ensure that your readers can learn or accomplish something.
```
An attempt to make explicit why thinking about audience helps readers learn.

* **Goal:** Find a solution to a low Lighthouse score that is directly applicable to her existing project as quickly as possible.

### Goal-driven problem solver (Damir)

> Damir is a developer at a small fintech startup. His boss has asked him to help turn their company's site into a PWA. After trying to enable some offline functionality, Damir realizes through a StackOverflow post ([for example, a post about PWA offline capability on data that frecuently changes ](https://stackoverflow.com/questions/50777677/pwa-offline-capability-on-data-that-frecuently-changes)) that the best way to solve his problem is to create a service worker. He needs to learn how to create a service worker that works with his company's existing site and the dynamic data that it handles.

Reviewer 2: ```suggestion
Damir is a developer at a small fintech startup. His boss has asked him to help turn their company's site into a PWA. After trying to enable some offline functionality, Damir realizes through a StackOverflow post ([for example, a post about whether highly dynamic sites should add PWA functionality](https://stackoverflow.com/questions/50777677/pwa-offline-capability-on-data-that-frecuently-changes)) that the best way to solve his problem is to create a service worker. He needs to learn how to create a service worker that works with his company's existing site and the dynamic data that it handles.
```
The actual title of that SO question was kinda confusing.

* **Goal:** Learn about the most recent web development tools and best practices to create a personal site from scratch to apply for industry jobs.

## Writing to the audience

> When you're writing a post or codelab, keep the web.dev personas in mind (or more, if their needs overlap). How do you do that in a given piece, especially when the needs of different readers compete? One way to address multiple audience segments is to provide multiple paths for a given topic. For example, you might write:

Reviewer 2: This changes the meaning. The intent was to say (a) target a specific persona, but (b) also be mindful of how to serve audience members not represented by that persona. Kate's doing a larger overhaul of this page, so let's hold off on further fiddling until she has a draft :)

* A post using vanilla JavaScript, with associated codelabs using common frameworks to solve the same problem
* A post about optimizing images in general, with associated codelabs showing how to integrate image optimization into different build processes
* A post about how to address an issue most web developers encounter, with sections at the end focused on solving the problem as it appears in particular business verticals

Be mindful of your [voice](/handbook/voice). You want readers to feel they can trust the guidance you're giving. They should feel neither talked down to nor overwhelmed. Check out [Instruction for adult learners](/handbook/effective-instruction#instruction-for-adult-learners) in the [Writing effective instruction](/handbook/effective-instruction) post.

### Value propositions

> A value proposition is a promise to readers that a piece of content will be useful and provide value to them. Create a value proposition for everything you write by crafting titles, subtitles, and descriptions that tell your target audience why your post or codelab is worth reading. Here are some common strategies:

Reviewer 2: ```suggestion
A value proposition is a promise to readers that a piece of content will be useful to them. Create a value proposition for everything you write by crafting titles, subtitles, and descriptions that tell your target audience why your post or codelab is worth reading. Here are some common strategies:
```
I was thinking that "be useful" and "provide value" are equivalent here. The goal of the aside was to connect the phrasing of the key term to its definition. If you don't think that's needed, I'd just cut it altogether to avoid redundancy.

> * State what readers will learn or understand by reading the piece

Reviewer 2: ```suggestion
* State what readers will learn or understand by reading the piece.
```

> * Explain what a piece helps readers do

Reviewer 2: ```suggestion
* Explain what a piece helps readers do.
```

> * Describe how a technique or tool improves the user experience

Reviewer 2: ```suggestion
* Describe how a technique or tool improves the user experience.
```
